src/i18n/config.json

```diff
@@ -113,9 +113,18 @@
     "name": "Brazilian Portuguese",
     "langDir": "ltr",
     "dateFormat": "DD.MM.YYYY",
-    "hrefLang": "",
+    "hrefLang": "pt-br",
     "enabled": false
   },
+  {
+    "code": "pt-pt",
```

Comment:
If it is a generic `pt` language, it should just be `pt` not `pt-pt` 🤔

Reply:
@ovflowd it was what I was trying to explain to @nschonni.

I changed it just to permit the pull request to be merged and
I could adjust in the future.

Reply:
For me, it's OK to keep `pt` up to you. I can merge it regardless :)